pulser-simulation/pulser_simulation/simulation.py

```diff
@@ -970,8 +972,17 @@ def _run_solver() -> CoherentResults:
                     progress_bar=p_bar,
                     options=solv_ops,
                 )
+            results = [
+                QutipResult(
+                    tuple(self._qdict),
+                    self._meas_basis,
+                    state,
+                    self._meas_basis == self.basis_name,
+                )
+                for state in result.states
+            ]
```

Comment:
I guess this one will also be part of the Qutip backend, or at least that's the idea, right?